Update reference for version 3.0

robnee opened the pull request:
This closes #22.  It also adds some missing documentation for several other new or improved features of SolveSpace 3.0.  I'm sure the wording could be improved in places so I would appreciate feedback

robnee commented:
@phkahler @ruevs These CI errors seem to be false positives I think but not sure.

ruevs commented on ref.pl:
```diff
@@ -349,6 +349,15 @@
 
+<p>
+    In order to reduce distraction when sketching in 2D, solid models from
+    inactive groups are "dimmed" so that they blend into the background.  To
```
```suggestion
    inactive groups are "dimmed" (by rendering them using the #def-dim-solid style) so that they blend into the background.  To
```

ruevs commented on ref.pl:
```diff
@@ -349,6 +349,15 @@
+<p>
+    In order to reduce distraction when sketching in 2D, solid models from
+    inactive groups are "dimmed" so that they blend into the background.  To
+    disable this uncheck &rarr; Darken Inactive Solids user.  The intensity of
```
```suggestion
    disable this uncheck &rarr; Darken Inactive Solids user.  
    By modifying the #def-dim-solid style any color may be assigned to geometry
    from inactive groups instead of just "dimming" them.
```

ruevs commented on ref.pl:
```diff
@@ -1483,8 +1516,8 @@
-be modified. These modifications will be saved in the registry, and will
-apply to all files opened on that computer.
+be modified. These modifications will be saved in the user config (not in the
+.slvs file), and will apply to all files opened on that computer.
```
```suggestion
These modifications will be saved in the user config (the regisrtry on Windows,
a .json file on other platforms), and will apply to all files opened on that computer.
```

ruevs commented on ref.pl:
```diff
@@ -1866,25 +1903,28 @@
 
 <p>
-    Two lights are available. If only one is desired, then the second
+    Two direction lights are available. If only one is desired, then the second
```
```suggestion
    Two directional lights are available. If only one is desired, then the second
```

ruevs reviewed (changes requested):
The CI failures are not critical. @jwesthues commented on this here: https://github.com/solvespace/solvespace-web/pull/20#issuecomment-719788553

ruevs commented:
@robnee This did not work out as expected. I did not realize that one can merge the suggestions directly and I did not make mine "strictly correct" since I do not know how to suggest multi-line changes.

Now it is a bit mixed up...

robnee commented:
@ruevs sorry about that.  I guess the hard line breaks make simply accepting the changes via GitHub a little flakey.  I think I've cleaned this up now and incorporated your suggestions (Thanks for those!)

ruevs commented:
Thanks for cleaning it up. The changes look good to me so I've merged them in.

@phkahler I think we should ask Jonathan to make this live on the site only after [3.0 is released](https://github.com/solvespace/solvespace/issues/772#issuecomment-818882057) together with the other changes that will be done (e.g. the download page).
